### DISCUSSION TITLE
Embed search components inside my webpart

### DISCUSSION THREAD

**Discussion by mpamaral72** [microsoft-search/pnp-modern-search#1497] [Q&A] [answered]
I am creating a webpart with React, in the end I will have a package that contains my entire application. Is it possible to embed search components inside my webpart? Any ideas for that?

**Comment by FranckyC** [comment, marked as answer]
Hi @mpamaral72, you can't really embed the search components "as is" in your app. You could try to reuse React components from he solution to integrate with yours.
